diskplorer.py

```diff
@@ -166,8 +169,8 @@ def fix_y_axis(plt):
 plt.savefig(fname=output_filename)
 
 with open(raw_filename, 'w') as raw:
-    print('buffersize,concurrency,iops,lat_avg,lat_05,lat_95,lat_99,max', file=raw)
-    for concurrency, iops, lat_avg, lat_05, lat_95, lat_99,lat_max in zip(
-            concurrencies, iopses, latencies, latencies_05, latencies_95, latencies_99, latencies_max):
-        print('{buffer_size},{concurrency},{iops},{lat_avg},{lat_05},{lat_95},{lat_99},{lat_max}'
+    print('buffersize,concurrency,iops,lat_avg(us),lat_05(us),lat_95(us),lat_99(us),lat_9999(us),lat_max(us)', file=raw)
```

Comment:
Latencies are in `ns`. In order to print them in `us` you need to divide them by 1000.